requirements/requirements.in

```diff
@@ -51,6 +55,7 @@ python-dsv-sdk>=1.0.4
 python-tss-sdk>=1.2.1
 python-ldap
 pyyaml>=6.0.1
+pyzstd
```

Comment:
Can you leave a comment about what this is here for? I can take it as self-obvious what the "opentelementry" requirements are for, but this would be confusing to see.